lib/screens/productsNew/newUI.dart

```diff
@@ -22,14 +22,14 @@ import 'package:shimmer/shimmer.dart';
 
 class Category {
   final String name;
-  final String resname;
+  final String categoryname;
```

Comment:
need this in camel case. Make sure you follow it throughout

Reply:
done